### PR TITLE
Add some details for non visible scrollspy elements

### LINKED ISSUE #36615: Scrollspy does not function within nav tabs
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

I am attempting to use Scrollspy within nav-tabs. Scrollspy _does_ work on the first tab of the nav tabs, but _not_ the second tab, or any other tabs. See the codepen for an example. 

### Reduced test cases

https://codepen.io/chrisrodkey/pen/PoQvOzQ

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome, Safari

### What version of Bootstrap are you using?

v5.2 beta 1

### PR DESCRIPTION
**Pull request by GeoSot**
closes #36615

It probably needs proofreading, from a native speaker 

Preview : https://deploy-preview-36625--twbs-bootstrap.netlify.app/docs/5.2/components/scrollspy/#how-it-works

### CODE REVIEW

**Comment by arcanisgk** [comment]
hi, i have try to test ScrollSpy in code pen https://codepen.io/arcanisgk-the-sasster/pen/dymYvor and it doesn't work correctly or I'm missing something to put in the html code...

The problem is related to:

on the Boostrap Page: clicking on a Menu item causes a graphical effect that moves the active menu item that was clicked on, thus exchanging the active items.

in codepen, the event in which I click on an element in the menu does not occur, and the element that is clicked does not remain active.

File: site/content/docs/5.2/components/scrollspy.md
Comment (julien-deramond) [inline site/content/docs/5.2/components/scrollspy.md:29]:
```suggestion
  <summary class="text-muted">Show tab example</summary>
```

In order to be consistent with what's done in _introduction.md_: https://github.com/twbs/bootstrap/blob/c1813ef2bc2a398e5a9bcdd398608f8a021574e3/site/content/docs/5.2/getting-started/introduction.md?plain=1#L89-L105

File: site/content/docs/5.2/components/scrollspy.md
Comment (julien-deramond) [inline site/content/docs/5.2/components/scrollspy.md:32]: Extra line to remove to avoid an extra top padding.

File: site/content/docs/5.2/components/scrollspy.md
Comment (julien-deramond) [inline site/content/docs/5.2/components/scrollspy.md:26]:
```suggestion
As of above, when the wrapper becomes visible (its elements too), you must force scrollspy instance to check again for observable elements, using the `refresh` method.
```

File: site/content/docs/5.2/components/scrollspy.md
Comment (julien-deramond) [inline site/content/docs/5.2/components/scrollspy.md:44]: Extra line to remove.

File: site/content/docs/5.2/components/scrollspy.md
Comment (julien-deramond) [inline site/content/docs/5.2/components/scrollspy.md:25]: "not visible" -> non-visible or invisible. I'll let a native speaker fix it :)

File: site/content/docs/5.2/components/scrollspy.md
Comment (julien-deramond) [inline site/content/docs/5.2/components/scrollspy.md:34]:
```suggestion
  el.addEventListener('shown.bs.tab', () => {
```

**Review by julien-deramond — COMMENTED** [review]
Few minor suggestions to enhance this warning.

**Comment by GeoSot** [comment]
> [arcanisgk](/arcanisgk) Hi, I tried to test ScrollSpy on codepen....

Please @arcanisgk as your issue, it is out of scope on this PR, try to remove it, and start a new discussion or issue

**Review by julien-deramond — APPROVED** [review]
LGTM! Let's wait for the review of a native speaker before merging it :)

**Comment by julien-deramond** [comment]
> On mobile for now, but how's this read?

Committed your proposal to save you some time with the "Co-authored-by" info.

**Review by mdo — APPROVED** [review]

File: site/content/docs/5.2/components/scrollspy.md
Comment (GeoSot) [inline site/content/docs/5.2/components/scrollspy.md:19]: @mdo as you transferred the whole section, maybe we should keep here a reference like :

'- Target elements that aren't visible will be ignored'

**Pull request merged**
